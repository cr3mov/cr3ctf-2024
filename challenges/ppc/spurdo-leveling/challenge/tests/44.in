115638964924336596 922800661973584150
2 9488
3 4983
