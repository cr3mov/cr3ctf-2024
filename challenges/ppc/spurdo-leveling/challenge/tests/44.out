55774025500916 55774025500916 55774025500916
